medication_management/ticket.py

```diff
@@ -1,7 +1,43 @@
-from medication_management.medicamento import Medicamento
+import locale
+from datetime import datetime
+from dataclasses import dataclass, field
+from typing import List
+from .medicamento import Medicamento
 
+@dataclass
 class Ticket:
-    
-    def __init__(self):
-        self.medicamentos = []
-        self.fecha = None
+    cliente: str = None
+    fecha: datetime = None
+    medicamentos: List[Medicamento] = field(default_factory=list)
+    totale: float = None
+
+    def cargar_ticket_desde_txt(self, file_path: str):
```

Comment:
La "S" di "SOLID" è "separation of concerns". Una funzione deve fare una cosa e soltanto una.
Questa legge e poi fa l'analisi globale, e per ogni riga. La "S" è molto importante, perchè il suo compito è fare possibile generare funzioni che si possono testare independentemente.

Reply:
Procederò a separare le responsabilità della funzione cargar_ticket_desde_txt, rendendola più modulare e facilmente testabile